src/energyplus/Test/LoadProfilePlant_GTest.cpp

```diff
@@ -0,0 +1,51 @@
+/***********************************************************************************************************************
+*  OpenStudio(R), Copyright (c) Alliance for Sustainable Energy, LLC.
+*  See also https://openstudio.net/license
+***********************************************************************************************************************/
+
+#include <gtest/gtest.h>
+#include "EnergyPlusFixture.hpp"
+
+#include "../ForwardTranslator.hpp"
+
+#include "../../model/Model.hpp"
+#include "../../model/LoadProfilePlant.hpp"
+#include "../../model/PlantLoop.hpp"
+
+#include <utilities/idd/LoadProfile_Plant_FieldEnums.hxx>
+
+#include <utilities/idd/IddEnums.hxx>
+#include "../../utilities/idf/IdfObject.hpp"
+#include "../../utilities/idf/IdfObject_Impl.hpp"
+
+#include "../../utilities/idf/WorkspaceObject.hpp"
+#include "../../utilities/idf/WorkspaceObject_Impl.hpp"
+
+using namespace openstudio::energyplus;
+using namespace openstudio::model;
+using namespace openstudio;
+
+TEST_F(EnergyPlusFixture, ForwardTranslator_LoadProfilePlant) {
+  Model m;
+
+  PlantLoop pl(m);
+  LoadProfilePlant lpp(m);
+  EXPECT_TRUE(pl.addDemandBranchForComponent(lpp));
+
+  EXPECT_TRUE(lpp.setPeakFlowRate(0.005));
```

Comment:
Nitpick, but I think we should test all fields while we're at it. Couple of extra lines only